src/main/java/Duke.java

```diff
@@ -6,5 +9,68 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        try (Scanner scan = new Scanner(System.in)) {
```

Comment:
Please follow the Java coding standards when using _exceptions_.
```suggestion
        try {
            Scanner scan = new Scanner(System.in)
```